tests/acceptance/pre-suite/02_chocolatey_application_install.rb

```diff
@@ -0,0 +1,35 @@
+require 'master_manipulator'
+test_name 'MODULES-3043 - C97739 - Install Client on Virgin System'
+
+chocolatey_pp = 'include chocolatey'
```

Comment:
This could be a failure point. I don't think we want to have this. I'd much prefer we use our internal chocolatey.nupkg that we use for our internal build systems.

Other than that, this looks pretty good.